api/src/main/resources/workbench-api.yaml

```diff
@@ -7101,6 +7104,9 @@ definitions:
       - lastModifiedEpochMillis
       - adminLocked
     properties:
+      recentlyModifiedId:
```

Comment:
This is only used for recent resources and is null otherwise. Does it seem ok to have this here, or should it be a different type?